pkg/verifier/codeowner_test.go

```diff
@@ -466,13 +466,13 @@ func TestValidateCodeownerFileGitlab(t *testing.T) {
 		./folder2/folder3/file3
 		./file4
 	*/
-	folder1 := filet.TmpDir(t, "")
-	folder2 := filet.TmpDir(t, "")
-	folder3 := filet.TmpDir(t, folder2)
+	folder1 := filet.TmpDir(t, "./")
+	folder2 := filet.TmpDir(t, "./")
+	folder3 := filet.TmpDir(t, "./"+folder2)
```

Comment:
Since we are talking about directories, the best approach is to use the `path/filepath` module function to append strings:

```golang
filepath.Join("./", folder2)
```